_layouts/home.html

```diff
@@ -18,73 +18,58 @@ <h1>Five days of talks, sprints, and tutorials in Durham, North Carolina</h1>
         </span>
       </div>
 
-      {% comment %}
       <a class="button" href="{{ site.ticket_link }}">Buy Tickets</a>
       <a class="button" href="{{ site.cfp_application }}">Submit Talk</a>
-      {% endcomment %}
 
-      <a class="button" href="https://emails.djangocon.us/sendy/subscription?f=Gs892S6cdJ676370nkSBxLOWzUsgYKPBcwkEMMvt0rr70f3IeA7634kuMTKJCLrdcgOQHQHdzwqLwvmcETbcm6KWtu9g">Join Mailing List</a>
       {% comment %}
+      <a class="button" href="https://emails.djangocon.us/sendy/subscription?f=Gs892S6cdJ676370nkSBxLOWzUsgYKPBcwkEMMvt0rr70f3IeA7634kuMTKJCLrdcgOQHQHdzwqLwvmcETbcm6KWtu9g">Join Mailing List</a>
       <a class="button" href="/schedule/">Conference Schedule</a>
       {% endcomment %}
     </div>
   </div>
 </header>
 
 
-<div class="home-content">
 
-<section class="section-pad curve1 theme-brand-color3">
-  <div class="row column xlarge-9">
-    <p class="lead column large-9 large-centered text-center">Join us for a source of inspiration, education and networking opportunities. Our team is making sure that participating in DjangoCon US is easy for you!</p>
-
-    <div class="date-card medium-4 column">
-      <a href="/tutorials/">
-        <div class="dates">
-          <div class="month">October</div>
-          <div class="days">15</div>
-        </div>
-        <div class="details">
-          <h2 class="date-card-title">Tutorials</h2>
-          <p>One day, numerous tutorials</p>
-        </div>
-      </a>
-    </div>
-    <div class="date-card medium-4 column">
-      <a href="/talks/">
-        <div class="dates">
-          <div class="month">October</div>
-          <div class="days">16-18</div>
-        </div>
-        <div class="details">
-          <h2 class="date-card-title">Talks</h2>
-          <p>Dozens of talks chosen by the community</p>
-        </div>
-      </a>
-    </div>
-    <div class="date-card medium-4 column">
-      <a href="/sprints/">
-        <div class="dates">
-          <div class="month">October</div>
-          <div class="days">19-20</div>
-        </div>
-        <div class="details">
-          <h2 class="date-card-title">Sprints</h2>
-          <p>Team up to work on Django!</p>
-        </div>
-      </a>
-    </div>
+<div class="row column xlarge-9 theme-brand-color5 section-pad">
+  <div class="date-card medium-4 column">
+    <a href="/tutorials/">
+      <div class="dates">
+        <div class="month">October</div>
+        <div class="days card-bg-1">15</div>
```

Comment:
Sorry if this is a known issue, but tutorials are the week before this year and are online only. Oct 8th